derive/src/rpc_attr.rs

```diff
@@ -73,7 +73,7 @@ impl RpcMethodAttribute {
 									.map_or(Vec::new(), |ml| get_aliases(ml));
 								Ok(RpcMethodAttribute {
 									attr: attr.clone(),
-									name: name.into(),
+									name: name,
```

Comment:
```suggestion
									name,
```

Reply:
There are more opportunities for such shorthand syntax in the codebase (about 20 in total, I guess) — but I decided to keep this focused on the non-style-related improvements only.

Reply:
True but if you're modifying this line anyway in a way that now allows the shorthand, I say just use it.